modules/backend/lang/nl/lang.php

```diff
@@ -355,6 +361,8 @@
         'permissions' => 'De map :name of de submappen zijn niet schrijfbaar voor PHP. Zet de bijhorende rechten voor de webserver in deze map.',
         'extension' => 'De PHP extensie :name is niet geïnstalleerd. Installeer deze bibliotheek en activeer de extensie.',
         'plugin_missing' => 'De plugin :name is een afhankelijkheid maar is niet geïnstalleerd. Installeer deze plugin a.u.b.',
+        'debug' => 'Debug modus is ingeschakeld. Dit wordt niet aanbevolen op een productie-omgeving.',
+        'decompileBackendAssets' => 'Assets van de back-end worden momenteel  gedecompileerd. Dit wordt aanbevolen op een productie-omgeving.',
```

Comment:
Remove double space between "momenteel" and "gedecompileerd" and change the last sentence to "Dit wordt niet aanbevolen op een productie-omgeving."

Reply:
@jacobdekeizer please don't play `an admin`, even though the comment was not related to PR.
But I was about to add a comment for removing double `space`. Thank you!

Reply:
@w20k Sorry, I will write reviews differently in the future :)